epub33/explainers/EPUB33-explainer.md

```diff
@@ -0,0 +1,134 @@
+# EPUB 3.3 Explainer
+
+**Written**: 2021-02-22, **Updated**: 2021-04-29, Duke Ellington’s birthday. 
+
+## Authors:
+
+ * Dave Cramer (Invited Expert)
+ 
+## Participate
+
+ * [Issue Tracker](https://github.com/w3c/epub-specs/issues)
+ * [Mailing List](https://lists.w3.org/Archives/Public/public-epub-wg/)
+ 
+## Table of Contents
+
+TK
+ 
+## Introduction
+
+**EPUB 3.3** is a packaging mechanism for web content, designed for electronic books. EPUB's roots go back to 1999, with the OEB 1.0 specification from the Open Ebook Forum (which evolved into the IDPF) In 2007, a packaging format was introduced and OEB became EPUB 2.0. In 2011, EPUB 3.0 added support for XHTML5. Changes since then have been minor. The IDPF was absorbed into W3C in 2017, and responsibility for the maintenance of EPUB fell to the EPUB 3 Community Group, which released EPUB 3.2 in 2019 as a community group note. EPUB is twenty-one years old, but has never gone through the W3C Recommendation track, until now. 
+
+EPUB 3.3 includes the following specifications:
+
+* [EPUB 3.3 Overview](https://w3c.github.io/epub-specs/epub33/overview/), a non-normative overview of EPUB 3.3
+* [EPUB 3.3 Core](https://w3c.github.io/epub-specs/epub33/core/), the specification of the file format
+* [EPUB 3.3 Reading Systems](https://w3c.github.io/epub-specs/epub33/rs/), the specification for user agents, known as EPUB Reading Systems. 
+* [EPUB Accessibility 1.1](https://w3c.github.io/epub-specs/epub33/a11y/)
+* [EPUB Accessibility Techniques 1.1](https://w3c.github.io/epub-specs/epub33/a11y-tech/)
+
+## Goals
+
+## Non-goals
+
+
+## Security and Privacy Questionnaire
+
+EPUB files are presented to end users by EPUB Reading Systems, which can be constructed as web applications but are more likely to be native apps. The EPUB specifications define the file format in great detail, but generally give user agents wide latitude, subject mostly to the requirements of the embedded technologies, such as HTML, SVG, XML, and CSS. 
+
```

Comment:
Maybe worth emphasizing what I said above: that reading systems are mostly based on Blink/Webkit, that EPUB does not add new APIs, does not fork the Web, etc. 

(We have to be slightly defensive here...)

Reply:
Note this is moot, as I forgot to delete this text when I split the explainer and the security questionnaire.